tests/playwright/specs/leadership.spec.js

```diff
@@ -21,9 +21,7 @@ test.describe(
         });
 
         test('Open / close biography', async ({ page }) => {
-            const leader = page.locator(
-                '#executive .vcard.has-bio:first-child'
-            );
+            const leader = page.locator('#laura-chambers.vcard.has-bio');
```

Comment:
I've updated the test here so it should pass for both the hard-coded page and the CMS page (it assumes at least our CEO will have data entered). So, things should pass OK after merging still because we're using `prefers_cms`, right?

Reply:
Yeah, once the page is live in the environment where playwright runs, I can see that it should pass, yeah.